salesforce_bulk/tests/test_salesforce_bulk.py

```diff
@@ -288,6 +288,52 @@ def test_query(self):
             ['Email', 'Id', 'Name']
         )
 
+    def test_query_pk_chunk(self):
+        bulk = SalesforceBulk(self.sessionId, self.endpoint)
+        self.bulk = bulk
+
+        job_id = bulk.create_query_job("Contact", contentType=self.contentType, pk_chunking=True)
```

Comment:
Maybe we should set the chunk size in relation to the number of Contact records in the org to ensure that we have more than 1 chunk?

Reply:
The number of chunks doesn't really matter since the thing we really need to test is that we chuck the original batch (which goes into NotProcessed) and pick up some new batches.

Reply:
Ah, I see. This is fine, then.